plugins/modules/inventory_workflow_manager.py

```diff
@@ -1799,7 +1799,7 @@ def provision_wired_device_v1(self, device_ip, site_name, device_type):
             site_name (str): The name of the site where the device will be provisioned.
             device_type (str): The type of device being provisioned.
         Returns:
-            bool: True if provisioning is successful, False otherwise.
+            self (object): An instance of the class after the provision operation is performed.
```

Comment:
provision_wired_device_v2 is not returning anything. Can you please check that API as well?

Reply:
yes, I have checked and we can safely remove the return. and testing also done